src/routes/movies.py

```
@@ -370,117 +794,204 @@
-            },
-        },
-    },
-)
-async def update_movie(
-    movie_id: int,
-    movie_data: MovieUpdateSchema,
-    db: AsyncSession = Depends(get_db),
+@router.post("/genres/", tags=["Genres"], response_model=GenreSchema)
+async def create_genre(
+        genre_name: str,
+        db: AsyncSession = Depends(get_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)